cloudigrade/util/aws/ec2.py

```diff
@@ -384,4 +384,6 @@ def is_instance_windows(instance_data):
         bool: True if it appears to be windows, else False.
 
     """
-    return instance_data.get('Platform', '').lower() == 'windows'
+    return instance_data.get('Platform', '').lower() == 'windows' if \
+        isinstance(instance_data, dict) \
```

Comment:
In what specific cases will `instance_data` not be a `dict`? If we intend to play polymorphically here and handle multiple object types, we should update the docstring above.

Reply:
Updated the doc with more info on this.